tests/signed-off-by-test.sh

```diff
@@ -4,10 +4,9 @@ success="true"
 
 for commit in $(git cherry master | cut -d " " -f 2)
 do
-    git show -s $commit | grep -q 'Signed-off-by:'
-    if [ $? -ne 0 ]; then
+    if ! git show -s "$commit" | grep -q 'Signed-off-by:'; then
```

Comment:
I agree that this should be a separate commit, but was trying to keep this pull request to only one commit. Either way, this change is needed for shellcheck-test.sh to pass.